plugins/modules/mso_schema_site_anp_epg_domain.py

```diff
@@ -414,11 +479,27 @@ def main():
 
         if vlan_encap_mode:
             vmmDomainProperties["vlanEncapMode"] = vlan_encap_mode
+        
```

Comment:
not sure why additional white line is added, can be removed?

Reply:
I removed it in the old commit